src/Fixer/ClassNotation/FinalInternalClassFixer.php

```diff
@@ -174,6 +179,25 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setAllowedTypes(['bool'])
                 ->setDefault(false)
                 ->getOption(),
+            (new FixerOptionBuilder('attribute_exclude', 'Class level attributes that must be omitted to fix the class, even if all of the white list ones are used as well. (case insensitive)'))
```

Comment:
I started to wonder - what's the need of having `annotation_exclude` and `attribute_exclude` both?
Maybe we should simply have `exclude` (alias for `annotation_exclude`) that is considered for both, annotations and attributes?

 It's like PHP introduced attributes and now ppl are migrating from annotations to use attributes, but the purpose and reasoning is the same, so maybe we don't need to distinguish them on config level?

Reply:
nice idea, not sure how to do it in a BC way just yet, need to think on it some more